tools/autograd/gen_variable_type.py

```diff
@@ -849,7 +849,9 @@ def gen_variable_type_func(
             if not fn.info:
                 key = "Default"
                 type_definition = METHOD_DEFINITION.substitute(
-                    return_type=cpp.returns_type(f.func.returns).cpp_type(),
+                    return_type=cpp.returns_type(
```

Comment:
this will generate `autograd::Node` functions that remember `vector<SymInt>`?

Reply:
Not quite, this is for the situation where a function returns a SymInt (e.g., `aten::size(Tensor) -> SymInt`), which is not really a thing right now